spyder/plugins/ipythonconsole/utils/kernelspec.py

```diff
@@ -9,6 +9,7 @@
 """
 
 # Standard library imports
+from packaging.version import parse
```

Comment:
`packaging` is not a standard library module. So, please move this import to the third-party section-